packages/snap-preact-components/src/components/Molecules/Slideout/Slideout.tsx

```diff
@@ -1,5 +1,5 @@
 /** @jsx jsx */
-import { h, Fragment, ComponentChildren } from 'preact';
+import { h, Fragment, ComponentChildren, cloneElement } from 'preact';
```

Comment:
Instead of using `cloneElement`, let's use `cloneWithProps` which is already being imported here.